README.md

```diff
@@ -224,6 +224,104 @@ messages with an unrecognised conversation ID on a dead letter queue.
 
 An example daisy chaining environment is provided in test-suite/daisy-chaining 
 
+## Endpoints
+
+The Patient Switching adaptors facade exposes two endpoints.
+
+- POST /Patient/$gpc.migratestructuredrecord
+- POST /$gpc.ack
+
+### /Patient/$gpc.migratestructuredrecord
+
+The migratestructuredrecord endpoint is the primary endpoint for the adaptor and is used to start an electronic health record transfer.
+The following is required to call this endpoint...
+
+- TO_ASID : The ASID identifier of the winning New Market Entrant (NMW)
```

Comment:
Shouldn't the facade be sending the request to the losing practice?

Reply:
Correct and corrected.